src/main/clojure/com/github/ericdallo/clj4intellij/test.clj

```diff
@@ -0,0 +1,47 @@
+(ns com.github.ericdallo.clj4intellij.test
+  "Test utilities for clj4intellij"
+  (:import
+   [com.intellij.testFramework EdtTestUtil LightProjectDescriptor]
+   [com.intellij.testFramework.fixtures CodeInsightTestFixture IdeaTestFixtureFactory]
+   [com.intellij.util ThrowableRunnable]
+   [com.intellij.util.ui UIUtil]))
+
+
+(defn setup
+  "Setup fixture factory, and return an instance of CodeInsightTestFixture
+   
+   ref: https://github.com/JetBrains/intellij-community/blob/2766d0bf1cec76c0478244f6ad5309af527c245e/platform/testFramework/src/com/intellij/testFramework/fixtures/CodeInsightTestFixture.java"
+  ^CodeInsightTestFixture []
+  (let [factory (IdeaTestFixtureFactory/getFixtureFactory)
+        raw-fixture (-> factory
+                        (.createLightFixtureBuilder LightProjectDescriptor/EMPTY_PROJECT_DESCRIPTOR (str *ns*))
```

Comment:
will this `*ns*` point to the caller instead of this one or should we receive that as arg?

Reply:
ohh missed this! Thanks, I'll change to receive as a parameter